src/styles/variables.scss

```
@@ -0,0 +1,4 @@
+$black: #060b35;
+$grey: #616070;
+$light-grey: #f3f3f3;
+$blue: #00acdc;
```

It is better to use more universal variables.
Example:
```suggestion
$main-color: #060b35;
$title-color: #616070;
$second-color: #f3f3f3;
$button-color: #00acdc;
```